src/transformers/utils/deprecation.py

```diff
@@ -0,0 +1,91 @@
+from functools import wraps
+from typing import Optional
+from warnings import warn
+
+import packaging.version
+
+from .. import __version__
+from . import ExplicitEnum
+
+
+class Action(ExplicitEnum):
+    RAISE = "raise"
+    NOTIFY = "notify"
+
+
+def warning(message):
+    warn(message, FutureWarning)
+
+
+def deprecate_kwarg(
+    old_name: str,
+    version: str,
+    new_name: Optional[str] = None,
+    raise_if_ge_version: bool = False,
```

Comment:
The motivation was to keep it short to keep the decorator in one line but probably will be rarely used since we prefer not to raise errors for deprecated args.